examples/line_sender.py

```diff
@@ -0,0 +1,19 @@
+from questdb.ilp import Sender
+
+if __name__ == "__main__":
+    with Sender(host='localhost', port=9009) as sender:
+        for _ in range(3):
+            sender.row(
+                'tab',
+                symbols={
+                    'name_a': 'val_a'
+                },
+                columns={
+                    'name_b': True,
+                    'name_c': 42,
+                    'name_d': 2.5,
+                    'name_e': 'val_b'
```

Comment:
Probably a separate example with also `at=` and a `datetime.datetime` timestamp.